src/components/DateNavigation.js

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import styled, {css} from 'styled-components';
+import {NavLink} from "react-router-dom";
+import Colors from "../constants/Colors";
+import {getMonthNameByIndex, isCurrentMonth} from "../utils/months";
+import {getBaseYear, getCurrentYear, getYears, isCurrentYear} from "../utils/years";
+
+class DateNavigation extends Component {
+    createUrl (month, year) {
+        return `${process.env.PUBLIC_URL}/stats/givers/${year}/${month}`
+    }
+    createPrevLink (currentMonth, currentYear) {
+        const month = currentMonth === 1 ? 12 : currentMonth - 1
+        const year = currentMonth === 1 ? currentYear - 1 : currentYear
+        return year < getBaseYear() ? '' : this.createUrl(month, year)
+    }
+    createNextLink (currentMonth, currentYear) {
+        const month = currentMonth === 12 ? 1 : currentMonth + 1
```

Comment:
What do you think about something more fancy? I know those below are less readable when you see them for the very first time , so you may consider those below as an interesting alternative
next:
![image](https://user-images.githubusercontent.com/3466388/50775383-514e6f80-1296-11e9-907c-0d492727b73b.png)
prev:
![image](https://user-images.githubusercontent.com/3466388/50775587-d6398900-1296-11e9-94ee-990e60b2a4a0.png)